cartography/intel/kubernetes/services.py

```diff
@@ -0,0 +1,85 @@
+import logging
+from typing import Dict
+from typing import List
+
+from neo4j import Session
+
+from cartography.intel.kubernetes.util import get_epoch
+from cartography.intel.kubernetes.util import K8sClient
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+
+
+@timeit
+def sync_services(
+    session: Session, client: K8sClient, update_tag: int, cluster: Dict, pods: List[Dict],
+) -> None:
+    services = list()
+    for service in client.core.list_service_for_all_namespaces().items:
+        item = {
+            "uid": service.metadata.uid,
+            "name": service.metadata.name,
+            "creation_timestamp": get_epoch(service.metadata.creation_timestamp),
+            "deletion_timestamp": get_epoch(service.metadata.deletion_timestamp),
+            "namespace": service.metadata.namespace,
+            "cluster_uid": cluster["uid"],
+            "type": service.spec.type,
+            "selector": service.spec.selector,
+            "load_balancer_ip": service.spec.load_balancer_ip,
+        }
+
+        ingresses = service.status.load_balancer.ingress
+        for ingress in ingresses or list():
+            item.update({"ingress_host": ingress.hostname, "ingress_ip": ingress.ip})
+
+        service_pods = list()
+        for pod in pods:
+            is_service_pod = True if service.spec.selector else False
+            for selector in service.spec.selector or dict():
+                if (
+                    selector not in pod["labels"] or
+                    service.spec.selector[selector] != pod["labels"][selector]
+                ):
+                    is_service_pod = False
+                    break
+            if is_service_pod:
+                service_pods.append(pod)
+        item["pods"] = service_pods
+        services.append(item)
```

Comment:
Code structure issue - 

Again, I think this is better suited in a `get_services()` function that gets called here.